src/js/pages/Arbitration/index.jsx

```diff
@@ -151,6 +151,12 @@ class Arbitration extends Component {
     return (
       <div className="escrow">
         <h2>Dispute Details <span className={"arbitrationStatus " + status}>{status}</span></h2>
+
+        {escrow.arbitration.result.toString() !== "0" && <Fragment>
```

Comment:
What do you think of using the constant ARBITRATION_UNSOLVED instead of "0"?  I probably have to change that in a couple of places too

Reply:
good idea. I'll do the changes